CONTRIBUTORS.md

```diff
@@ -0,0 +1,10 @@
+<!---
+Recommended contributor format
+
+* [Your Name or Github ID](link to Github profile)
+-->
+
+Thank you to all our contributors!
+
+* [John Qualls](https://github.com/Alton09)
+* [Aaron Alaniz](https://github.com/aaalaniz)
```

Comment:
```suggestion
* [John Qualls](https://github.com/Alton09)
* [Aaron Alaniz](https://github.com/aaalaniz)
* [Iñaqui Delgado](https://github.com/idelgado)
* [cyodrew](https://github.com/cyodrew)
* [Tejas Nandanikar](https://github.com/tejas-n)
```

Reply:
@Alton09 I left these out so that they could submit PRs with this change themselves so that they show up in the contributors page.